### DISCUSSION TITLE
Maquette du site

### DISCUSSION THREAD

**Discussion by NDOY3M4N** [Code-for-Senegal/etat#1] [Ideas]
Bonjour, bonsoir
Je me demandais si vous avez déjà une maquette pour le design du site. Je peux vous proposer des design si vous voulez. https://www.figma.com/file/wBZoRQ0q8Nl7pIHrFfp1Rz/Code-for-Senegal?node-id=0%3A1

**Comment by dofbi** [comment]
Hello @NDOY3M4N 🥇 
nice work, je paratge se [site](https://etat.public.lu/) avec toi pour rajouter des choses sur ton **mockup** hâte de voir la suite. encore merci pour la contribution 😃 

## Proposition 1
![wire1-home](https://user-images.githubusercontent.com/84446/111625272-93638c00-87e4-11eb-97d2-98770cccaa7c.png)

## Proposition 2
![wire2-home](https://user-images.githubusercontent.com/84446/111625601-f1906f00-87e4-11eb-8ee2-fc83086f5577.png)

**Reply by NDOY3M4N** [reply]
Super je vais essayer d'apporter les modifications nécessaires. Merci encore 🙏🏿

**Reply by NDOY3M4N** [reply]
Hello, 
j'avais oublié de vous demander votre avis sur le nouveau design des `card`. C'est ok ou bien ?
Maintenant pour la partie développement comment est-ce que ça va se passer ?

**Reply by dofbi** [reply]
Hello @NDOY3M4N 
avis ici #2
